org.remmina.Remmina-local.json

```diff
@@ -21,7 +21,7 @@
         "--share=ipc",
         "--share=network",
         "--socket=cups",
-        "--socket=fallback-x11",
+        "--socket=x11",
```

Comment:
Why was fallback-x11 removed? You almost always don't need both x11 and wayland

Reply:
With `fallback-x11`, on a Wayland system, environment variable `DISPLAY` was not being set in the flatpak, and this was causing keyboard input issues for RDP connections. I tried several ways to set `DISPLAY` during the flatpak build but none of them worked (including setting `--env=DISPLAY=:0` in the `finish-args`). I'm not sure if this is a bug with flatpak-builder. Switching `fallback-x11` to `x11` allowed for `DISPLAY` to be set which fixed the input issue. I tested on a Wayland and x11 desktop separately and have not seen any issues yet.

Reply:
No not a bug, display unset for wayland and fallback-x11 https://github.com/flatpak/flatpak/blob/2cb17b4eb82ecedaa98b5b7f954cf3e52fa95682/common/flatpak-run-sockets.c#L192-L197

You can't set it like that.

Thanks for the clarification, I'll approve the change.